pkg/vmimage/resources.go

```diff
@@ -148,8 +148,8 @@ func vm(subscriptionID, resourceGroup, location string, sshPublicKey string) *co
 				ImageReference: &compute.ImageReference{
 					Publisher: to.StringPtr("RedHat"),
 					Offer:     to.StringPtr("RHEL"),
-					Sku:       to.StringPtr("7-RAW"),
-					Version:   to.StringPtr("latest"),
+					Sku:       to.StringPtr("8"),
+					Version:   to.StringPtr("8.0.2019050711"),
```

Comment:
Is this going to need to change on every update?

Reply:
I think they will publish RAW image later on or "latest".